src/main/java/com/jcabi/manifests/Manifests.java

```diff
@@ -237,6 +260,7 @@ public MfMap append(final Mfs streams) throws IOException {
                     this.attributes.put(attr.getKey(), attr.getValue());
                     ++saved;
                 }
+                this.addToMultiMap(attr.getKey(), attr.getValue());
```

Comment:
@pinaf no, that if is to ignore attributes it has seen before. This PR is to collect all values for attributes, so it can't ignore values at this point here.